src/tests/ui/planner/workitem-quickpreview.ts

```diff
@@ -240,13 +241,19 @@ export class WorkItemQuickPreview extends ui.BaseElement {
 
   async updateDescription(description: string, append: boolean = false) {
     await this.descriptionDiv.clickWhenReady();
```

Comment:
do we need this line "await this.descriptionDiv.clickWhenReady()" here...please remove this